package.json

```diff
@@ -22,7 +22,8 @@
     "generate:definitions": "shx rm -rf generated/definitions && gen-api-models --api-spec openapi/index.yaml --out-dir generated/definitions",
     "generate": "npm-run-all generate:*",
     "preversion": "auto-changelog  --config .auto-changelog.json --unreleased --commit-limit false --stdout --template preview.hbs",
-    "version": "auto-changelog -p --config .auto-changelog.json --unreleased && git add CHANGELOG.md"
+    "version": "auto-changelog -p --config .auto-changelog.json --unreleased && git add CHANGELOG.md",
+    "postbuild": "npx dependency-check package.json --no-dev --missing ./dist/**/*.js"
```

Comment:
We can include `dependency-check` as a `devDependency` I guess

Reply:
Done in [4ea5745](https://github.com/pagopa/io-functions-cgn/pull/26/commits/4ea5745395b034ab9ef067f47f9d5f64596cf4f1)